orbit/cmd/orbit/orbit.go

```diff
@@ -303,6 +319,7 @@ func main() {
 		opt.ServerURL = c.String("update-url")
 		opt.LocalStore = localStore
 		opt.InsecureTransport = c.Bool("insecure")
+		opt.ServerCertificate = c.String("update-tls-certificate")
```

Comment:
Should that be `UpdateServerCertificate`? It could be a bit confusing with the Fleet server certificate.

Reply:
Nevermind, I understand that this is the `ServerCertificate` of the `update.Options`, so it is implied that this is the update server's certificate.